internal/charmstore/search.go

```diff
@@ -499,7 +499,7 @@ func createSearchDSL(sp SearchParams) elasticsearch.QueryDSL {
 			Modifier: "ln2p",
 		},
 		elasticsearch.BoostFactorFunction{
-			Filter:      ownerFilter(""),
```

Comment:
This filter was to boost promulgated entities, using promulgatedFilter just makes it clearer what is being boosted.